libbeat/reader/readjson/docker_json.go

```diff
@@ -136,6 +137,13 @@ func (p *DockerJSONReader) parseDockerJSONLog(message *reader.Message, msg *logL
 	message.AddFields(common.MapStr{
 		"stream": msg.Stream,
 	})
+
+	if len(msg.Attrs) > 0 {
+		message.AddFields(common.MapStr{
+			"attrs": msg.Attrs,
```

Comment:
I think this is quite docker specific, can you make it `docker.attrs`? We would also need to add an entry like this one for it: https://github.com/elastic/beats/blob/master/libbeat/processors/add_docker_metadata/_meta/fields.yml#L23-L27 and then run `make update`

Reply:
So the long story here is that these fields are shared between the `docker` input and `add_docker_metadata`. Most input fields are defined under `filebeat/_meta/fields.common.yml`, you could perhaps add it there?

About building, please report any issue you may have, we are here to help :)

Reply:
hi @exekias 
I have now
- changed the code so the fields get written to ```docker.attrs```
- added a test
- updated the changelog